README.md

```diff
@@ -502,6 +502,30 @@ When perfection is achieved and there is nothing left to `delete`, you can do th
 delete delete!
 ```
 
+## Forget
+
+You can forget variables which were declared, by using the `forget` keyword.
+
+```java
+const const name = "John Doe"!
+forget name! // Name has been forgotten
+print(name)! // Error/gibberish 
+```
+
+`forget` keyword works on variables declared with `const` and `const const`, but not variables declared with `const const const`
+
+```java
+const const const string = "C is the BEST LANGUAGE!!!"!
+forget string! // Does not work because const const const is immune to forget.
+print(string)! // Works
```

Comment:
Would attempting to forget a constant constant constant produce a runtime error or just quietly be ignored? The third line being run implies the ignored option, but clearly stating it would be pretty helpful in implementing the specification